app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -2518,23 +2550,13 @@ describe('NetworkController', () => {
       await withController(
         {
           state: {
-            providerConfig: {
-              type: 'rpc',
-              rpcUrl: 'https://mock-rpc-url-1',
-              chainId: '0x111',
-              ticker: 'TEST1',
-              nickname: 'test network 1',
-              rpcPrefs: {
-                blockExplorerUrl: 'https://test-block-explorer-1.com',
-              },
-            },
             networkConfigurations: {
               testNetworkConfiguration: {
-                id: 'testNetworkConfiguration',
-                rpcUrl: 'https://mock-rpc-url-2',
-                chainId: '0x222',
-                ticker: 'TEST2',
-                nickname: 'test network 2',
+                rpcUrl: 'https://mock-rpc-url',
```

Comment:
This test doesn't exist on the core side. I opted to leave it because I didn't know if we wanted to bring it over to the core side or not.

Reply:
This behavior isn't part of refresh network, so this is the right place for it either way.

Porting these tests seems worthwhile. `setActiveNetwork` isn't well tested on core at the moment.

Reply:
Tests ported here: Tests ported here: https://github.com/MetaMask/core/pull/1391